src/native/minipal/CMakeLists.txt

```diff
@@ -2,8 +2,11 @@ include(configure.cmake)
 
 set(SOURCES
     cpufeatures.c
+    guid.c
+    memory.c
     random.c
```

Comment:
cc @lambdageek is there anything in `src/mono/mono/utils/mono-rand.c` or elsewhere in mono runtime where we can reuse these implementations?